### PR TITLE
fix: project name from main to explorer

### PR DESCRIPTION
**Pull request by YasunoriMATSUOKA**
@mkXultra 
Plz review🙏 

- Rename: telescope(main) -> explorer
- Fix: build/ci/cd

### CODE REVIEW

File: projects/explorer/src/app/views/accounts/account/account.module.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { MaterialModule } from '../../../views/material.module';
+import { MaterialModule } from '../../material.module';
```
Comment (mkXultra) [inline projects/explorer/src/app/views/accounts/account/account.module.ts:1]: @YasunoriMATSUOKA 
Is it a deliberate fix?

**Review by mkXultra — CHANGES_REQUESTED** [review]

File: projects/explorer/editions/ununifi/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  ununifi-explorer:
+    container_name: ununifi-explorer
+    image: ghcr.io/ununifi/explorer
+    volumes:
+      - ./config.js:/usr/share/nginx/html/assets/config.js
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    # ports:
+    #   - 80:80
```
[the remark this answers is not in the document]
Comment (YasunoriMATSUOKA) [reply projects/explorer/editions/ununifi/docker-compose.yml:10]: @mkXultra 
I think it is opened via `network_mode: host` and `nginx.conf` mounted to the container.
Comments are left to share that we are using the target port.

File: projects/explorer/src/app/views/accounts/account/account.module.ts (hunk 1 above)
Comment (YasunoriMATSUOKA) [reply projects/explorer/src/app/views/accounts/account/account.module.ts:1]: @mkXultra 
It seems that this was automatically corrected when the directory was moved. The useless description in the original file path specification has been erased, but I thought it pointed to the same file path as a result and was fine with it as it was.

**Comment by mkXultra** [comment]
LGTM

**Review by mkXultra — APPROVED** [review]

**Comment by mkXultra** [comment]
@YasunoriMATSUOKA 
plz merge

**Pull request merged**